### ISSUE TITLE
[Feature request] GUI keyboard navigation?

### ISSUE THREAD

**Issue by Zzooouhh** [manjaro/manjaro-settings-manager#69]
I'm liking MSM, but one thing that seems missing to me for a truly smooth GUI experience is being able to control it via keyboard only. At the moment I'm using version 0.4.96-4 and the keyboard usability is very inconsistent between the different settings menus. On the initial screen for instance it is possible to select the individual menus like Language, User Accounts etc. but there seems to be no way to access them without a mouseclick. Similar case with the menus for kernel, language and language packages where it's impossible to tab-select the central menus, only the bottom (or top buttons).
